examples/Examples/LetRec.hs

```diff
@@ -5,6 +5,7 @@ module Examples.LetRec (tests) where
 
 import Plutarch (printTerm)
 import Plutarch.Builtin (pasConstr, pforgetData)
+import Plutarch.Internal.PlutusType (pcon', pmatch')
```

Comment:
Why not import it from `Plutarch`?

Reply:
Primarily since I meant for the import to be "more granular". But it's probably better to import `Plutarch` than an internal module - especially since it's a public facing example.